File: snippets/java/collection_exists.java
import static io.qdrant.client.ConditionFactory.matchKeyword;

import io.qdrant.client.QdrantClient;
import io.qdrant.client.QdrantGrpcClient;

QdrantClient client = new QdrantClient(QdrantGrpcClient.newBuilder("localhost", 6334, false).build());

client.collectionExistsAsync("{collection_name}").get();

File: snippets/go/collection_exists.go
package client

import (
	"context"
	"fmt"

	"github.com/qdrant/go-client/qdrant"
)

func collectionExists() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	exists, err := client.CollectionExists(context.Background(), "{collection_name}")
	if err != nil {
		panic(err)
	}
	fmt.Println("Collection exists: ", exists)
}
